docs/src/operator/monitoring.md

```diff
@@ -39,19 +15,22 @@ OpenTelemetry provides a
 [Span Metrics Converter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/connector/spanmetricsconnector)
 which can be used to convert our traces into more conventional metrics.
 
-### What gets traced
+## What gets traced
+
+We assign a unique trace (aka root span) to each RPC request, batch build, and block build process.
+
+<div class="warning">
 
-We assign a unique trace (aka root span) to each RPC request/response, batch build, and block build process.
+Span and attribute naming is unstable and should not be relied upon. This also means changes here will not be
+considered breaking, however we will do our best to document them.
 
-> [!CAUTION]
-> Span and attribute naming is unstable and should not be relied upon. This also means changes here will not be
-> breaking, however we will do our best to document them.
+</div>
 
-#### RPC request/response
+### RPC request/response
 
 Not yet implemented.
```

Comment:
I thought this has already been implemented?

Reply:
They are for the internal gRPC communication i.e. they're added as part of the block-producer's spans. However they haven't been added for requests arriving at the RPC component.